src/__tests__/__snapshots__/Organization-test.js.snap

```diff
@@ -188,7 +188,7 @@ exports[`ItemDetails for Organization renders correctly 1`] = `
                 alt={undefined}
                 className="portrait"
                 sizes={undefined}
-                src="https://s3.amazonaws.com/uploads.participedia.xyz/uis_favicon_0.png"
+                src="https://detetf146kq8c.cloudfront.net/uis_favicon_0.png"
```

Comment:
You should change your config to use `https://s3.amazonaws.com/uploads.participedia.xyz` instead of cloudfront, otherwise we're going to keep clobbering each other's changes every time we update snapshots (and your tests will fail in TravisCI).